api/integrations/launch_darkly/launch_darkly.py

```diff
@@ -0,0 +1,239 @@
+                )
+                FeatureStateValue.objects.create(feature_state=feature_state)
+
+    def _create_multivariate_feature(
+        self, ld_flag, environments: list[Environment], project: Project
+    ):
+        # todo try get tags
+        try:
+            feature = Feature.objects.create(
+                name=ld_flag.get("key"),
+                project=project,
+                initial_value=None,  # todo
+                description=ld_flag.get("description"),
+                default_enabled=None,  # todo
+                type=STANDARD,
+                tags=None,  # todo
+                is_archived=ld_flag.get("archived", False),
+                owners=None,  # todo
+                is_server_key_only=None,  # todo
+            )
+        except IntegrityError:
+            # Feature with this name already exists
+            self.logger.error(
+                f"Unable to create feature with name: {ld_flag.get('key')}"
+            )
+
+        feature_options = {}
+        for variant in ld_flag.get("variations"):
+            value = variant.get("value")
+            variant_type = self._get_multivariant_kind(value)
+            variant_values = self._get_multivariant_value(value, variant_type)
+            feature_option = MultivariateFeatureOption.objects.create(
+                feature=feature,
+                type=variant_type,
+                boolean_value=variant_values[0],
+                integer_value=variant_values[1],
+                string_value=variant_values[2],
+            )
+            feature_options[value] = feature_option
+
+        for environment in environments:
+            ld_environment = ld_flag.get("environments", {}).get(environment.name)
+            if ld_environment:
+                env_feature_option = feature_options.get(ld_environment.get(""))
+                MultivariateFeatureStateValue.objects.create(
+                    feature=feature,
+                    environment=environment,
+                    identity=None,  # todo
+                    feature_segment=None,  # todo
+                    enabled=ld_environment.get("_summary").get("on"),
+                    multivariate_feature_option=env_feature_option,
+                    updated_at=ld_environment.get("_summary").get("lastModified"),
+                )
+
+    def _get_multivariant_kind(self, value: any) -> str:
+        if isinstance(value, bool):
+            return BOOLEAN
+        if isinstance(value, str):
+            return STRING
+        if isinstance(value, float):
+            return FLOAT
+        if isinstance(value, int):
+            return INTEGER
+        return STRING
+
+    def _get_multivariant_value(
+        self, value: any, type: str
+    ) -> (Optional[bool], Optional[int], Optional[str]):
+        if type == BOOLEAN:
+            return (value, None, None)
+        if type in [INTEGER, FLOAT]:
+            return (None, int(value), None)
```

Comment:
We should perhaps log a warning here that we're possibly losing data if the value is a float.

Reply:
Absolutely will add it to the logs